src/fontra/client/core/html-utils.js

```diff
@@ -15,6 +16,7 @@ export class SimpleElement extends HTMLElement {
   }
 
   _attachStyles() {
+    this._appendStyle(shadowDomStyles);
```

Comment:
Shouldn't we be able to append a `<link>` element that references a "proper" CSS file instead? There is a "link" convenience function in this module.